src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -229,6 +233,31 @@ private function extractFromConstructor(string $class, string $property): ?array
         return null;
     }
 
+    private function extractFromDefaultValue(string $class, string $property)
+    {
+        try {
+            $reflectionClass = new \ReflectionClass($class);
+        } catch (\ReflectionException $e) {
+            return null;
+        }
+
+        $defaultValues = $reflectionClass->getDefaultProperties();
+
+        if (!isset($defaultValues[$property])) {
+            return null;
+        }
+
+        $map = array(
```

Comment:
What about storing this as a static property on the class instead?

Reply:
Sure. Actually, quite obviously.

Reply:
Can even be a `private const` as it targets master.